plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/confluent/ConfluentModule.java

```diff
@@ -66,7 +68,8 @@ protected void setup(Binder binder)
         configBinder(binder).bindConfig(ConfluentSchemaRegistryConfig.class);
         install(new ConfluentDecoderModule());
         install(new ConfluentEncoderModule());
-        binder.bind(ContentSchemaReader.class).to(AvroConfluentContentSchemaReader.class).in(Scopes.SINGLETON);
+        binder.bind(AvroConfluentContentSchemaReader.class).in(SINGLETON);
+        binder.bind(ContentSchemaReader.class).annotatedWith(ForKafkaRead.class).to(AvroConfluentContentSchemaReader.class);
```

Comment:
Why do you bind it twice? It looks like its bounded to the same class with same dependencies twice.